src/components/WorkflowChart/tasks.tsx

```diff
@@ -694,6 +720,13 @@ const pickTaskProperties = (
         inputParameters: task.inputParameters || {},
         type: task.type,
       } as WorkflowDefinition.IParallelTask;
+    case Task.TaskTypes.DynamicTask:
+      return {
+        taskReferenceName: task.taskReferenceName,
+        dynamicTasks: [],
```

Comment:
Should be
```typescript
{
  ...
  dynamicTasks: task.dynamicTasks || [],
  ...
}
```

To prevent dynamicTasks missing when user editing ``dynamicTask.dynamicTasks``

Reply:
nvm, No one gonna define the ``dynamicTasks`` at workflow definition, this only exist on runntime anyway.